src/main/javadoc/overview.html

```diff
@@ -5,6 +5,6 @@
     <title>Kitteh IRC Client Library</title>
 </head>
 <body>
-<p>Get started with Kitteh by visiting our build a client workshop in the {@link org.kitteh.irc.ClientBuilder} class.
+<p>Get started with Kitteh by visiting our build a client workshop in the {@link org.kitteh.irc.client.library.ClientBuilder} class.
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Let me know if you'd rather I removed this trailing NL. I think GitHub's editor added it